package.json

```diff
@@ -27,7 +27,7 @@
   },
   "devDependencies": {
     "@travi/any": "2.1.5",
-    "check-engine": "1.10.1",
+    "ls-engines": "0.5.1",
```

Comment:
fascinating that Renovate replaces dependencies, I wish it would link to more information with more context

Reply:
This particular replacement was [defined by me](https://github.com/form8ion/renovate-config/blob/fca0c141c21bc14f6f398303ba6b37e10c5d25e3/js-replacements.json#L22-27) for my preferences in my own projects after trying both of these and learning to prefer one over the other. So, I at least remember the context for myself, but I agree that it would be helpful for a team to have some sort of comment for those without the context. 

I've also contributed some definitions of name changes into renovate itself since those are a bit more official than just preferences.

Reply:
very cool!